saio/Vagrantfile

```diff
@@ -14,6 +14,7 @@
 #     will collide in the contents of the $GOPATH/bin (only executables, not libraries)
 
 Vagrant.configure(2) do |config|
+  config.ssh.forward_agent = true
```

Comment:
What does this do btw? Does this make it so I don't have to do:
`vagrant ssh -- -Y` to get my X Display forwarded or something?

Reply:
Oh... I see... this makes it so I don't have to do:
`vagrant ssh -- -A` to get my .ssh creds over there... Cool!

Reply:
Perhaps we also need this line then:
`config.ssh.private_key_path = "~/.ssh/id_rsa"`, hmm?

Reply:
Sorry to be so anal about this.. the web sure does have a lot of discussion about this stuff... all new to me... particularly w.r.t. the host being OSX (as it is for me). I can try out your branch on Monday if you'd like...